command/sub8_alarm/alarm_handlers/bus_voltage.py

```diff
@@ -0,0 +1,23 @@
+from ros_alarms import HandlerBase, Alarm
+import subprocess
+
+
+class BusVoltage(HandlerBase):
+    alarm_name = 'bus-voltage'
+
+    def __init__(self):
+        self.initial_alarm = Alarm(self.alarm_name, False, node_name='alarm_server')
+
+    def raised(self, alarm):
+        if alarm.severity == 3:
+            subprocess.Popen('echo "Warning: bus voltage is low!" | wall', shell=True,
```

Comment:
From a glance at the code, it looks like this will spam the terminal as the alarm is raised continually. Either on the thruster driver end or the alarm handler end it should probably only wall (or raise) once for each of the warning and killing voltage